cloudrun-malware-scanner/cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+steps:
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'build'
+      - '-t'
+      - "${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:$COMMIT_SHA"
+      - '-t'
+      - '${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:latest'
+      - '-f'
+      - 'cloudrun-malware-scanner/Dockerfile'
+      - 'cloudrun-malware-scanner'
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'push'
```

Comment:
what does this push command actually do?
isnt the previous command enough?

Reply:
The docker build is to build the docker image to in the server tmp local storage.
The docker push is to push this local image to the container registry.
I tested on the dev, it can't find the image without the push command, which is expected.